lib/matplotlib/rcsetup.py

```diff
@@ -354,15 +359,19 @@ def validate_aspect(s):
 
 
 def validate_fontsize(s):
+    fontsizes = ['xx-small', 'x-small', 'small', 'medium', 'large',
+                 'x-large', 'xx-large', 'smaller', 'larger']
     if isinstance(s, six.string_types):
         s = s.lower()
-    if s in ['xx-small', 'x-small', 'small', 'medium', 'large', 'x-large',
-             'xx-large', 'smaller', 'larger']:
+    if s in fontsizes:
         return s
     try:
         return float(s)
     except ValueError:
-        raise ValueError('not a valid font size')
+        raise ValueError("%s is not a valid font size. Valid font sizes "
+                         "are %s." % (s, ", ".join(fontsizes)))
+
+
 validate_fontsizelist = _listify_validator(validate_fontsize)
```

Comment:
PEP8 question... does the 2 blank line rule have to be right after the function, or can it wait until after ancillary actions like munging of docstrings and such?